pkg/cvo/upgradeable.go

```diff
@@ -135,14 +135,15 @@ type upgradeableCheck interface {
 
 type clusterOperatorsUpgradeable struct {
 	coLister configlistersv1.ClusterOperatorLister
+	operator *Operator
```

Comment:
I would prefer an interface or pass two objects vs passing a static type.  Keep this testable.